### PR TITLE
en,zh: optimize catalog structure of Restore Data into TiDB in Kubern… (#912)

### LINKED ISSUE #493: Improve the data import doc
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.
https://pingcap.com/docs/tidb-in-kubernetes/stable/restore-data-using-tidb-lightning/#deploy-tidb-lightning
Data Import

- Deploy tikv-importer
- Deploy tidb-lightening
  - Configure
    - SecretKey
      - Grant Permissions
      - Create Secret
    - IAM role
      - Grant Permissions
      - Create Secret
    - Service Account
      - Grant Permissions
      - Create Secret
  - Deploy
  - Destroy

2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

### LINKED ISSUE #494: Improve the title of data import doc
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.
https://pingcap.com/docs/tidb-in-kubernetes/stable/restore-data-using-tidb-lightning/#deploy-tidb-lightning
Change name to Import Data
Change document name from “Restore Data into TiDB in Kubernetes” to “Import Data”

2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #912 to release-1.1

---

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

fix #493 , change catalog structure of  `Deploy TiDB Lightning` section.
fix #494 , change docs title

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [x] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by DanielZhangQD** [comment]
@BinChenn Please help fix the conflict.

File: zh/restore-data-using-tidb-lightning.md [hunk 1]
```diff
@@ -117,22 +119,33 @@ helm inspect values pingcap/tidb-lightning --version=${chart_version} > tidb-lig
...
+        使用 Amazon S3、Ceph 或 GCS 作为存储后端时支持通过 AccessKey 和 SecretKey 授权。
 
-    2. 新建一个包含 rclone 配置的 `Secret`。rclone 配置示例如下。一般只需要配置一种云存储。有关其他的云存储，请参考 [rclone 官方文档](https://rclone.org/)。和使用 BR 和 Dumpling 进行数据恢复时一样，使用 Amazon S3 作为后端存储时，同样存在三种权限授予方式，参考[使用 BR 工具备份 AWS 上的 TiDB 集群](backup-to-aws-s3-using-br.md#aws-账号权限授予的三种方式)。在使用不同的权限授予方式时，需要使用不用的配置。
+        1. 新建一个包含 rclone 配置的 `Secret` 配置文件 `secret.yaml`。rclone 配置示例如下。一般只需要配置一种云存储。
 
+<<<<<<< HEAD
        + 使用 Amazon S3 AccessKey 和 SecretKey 权限授予方式，或者使用 Ceph、GCS 作为存储后端时:
 
+=======
+>>>>>>> e2e2aee... en,zh: optimize catalog structure of Restore Data into TiDB in Kubern… (#912)
```
Comment (ran-huang) [inline zh/restore-data-using-tidb-lightning.md:148]: @BinChenn Please resolve the conflict. Thanks~

**Comment by BinChenn** [comment]
PTAL @ran-huang

**Review by ran-huang — APPROVED** [review]
LGTM

**Pull request merged**